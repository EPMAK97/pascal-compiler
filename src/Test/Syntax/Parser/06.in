procedure foo(a, b : integer; c : array of integer);
type c = integer;

procedure bar;
const m = 0;
Begin
end;

Begin
   a := a - b;
end;
Begin

end.