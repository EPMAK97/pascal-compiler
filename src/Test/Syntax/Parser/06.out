└── Main Program
    └── procedure
        ├── integer
        │   ├── a
        │   └── b
        ├── array
        │   └── integer
        │       └── c
        ├── type
        │   └── integer
        │       └── c
        ├── procedure
        │   └── const
        │       └── m
        │           └── 0
        └── a
            └── -
                ├── a
                └── b
